templates/blocks/header.twig

```diff
@@ -1,38 +1,39 @@
 {% if ( header_title ) %}
-	<div class="clearfix page-content-wrap">
-		<div class="fixed-element clearfix">
-			<div class="container">
-				<div class="page-header col-md-8 col-lg-8 col-sm-12 pl-0 pr-0">
-					{% if ( issue or campaigns ) %}
-						<div class="top-tags-wrap">
-							{% if ( issue ) %}
-								<a class="category_link category-topic-title" href="{{ issue.link|default('#') }}">{{ issue.name|e('wp_kses_post')|raw }}</a>
-							{% endif %}
-
-							{% if ( campaigns ) %}
-								<span class="category-topic-title-seprater"></span>
-								{% for campaign in campaigns %}
-									<a href="{{ campaign.link }}">#{{ campaign.name|e('wp_kses_post')|raw }}</a>
-								{% endfor %}
-							{% endif %}
-						</div>
-					{% endif %}
-					<h1>{{ header_title|e('wp_kses_post')|raw }}</h1>
-					{% if ( header_subtitle ) %}
-						<h3>{{ header_subtitle }}</h3>
-					{% endif %}
-					{% if ( header_description ) %}
-						<p>{{ header_description|e('wp_kses_post')|raw }}</p>
+	<div class="page-header">
+		<div class="skewed-overlay"></div>
+		{% if ( background_image ) %}
+			<div class="page-header-background">
+				<img src="{{ background_image }}" class="page-header-image" />
+			</div>
+		{% endif %}
+		<div class="container">
+			{% if ( issue or campaigns ) %}
+				<div class="top-page-tags">
+					{% if ( issue ) %}
+						<a class="tag-item tag-item--main" href="{{ issue.link|default('#') }}">{{ issue.name|e('wp_kses_post')|raw }}</a>
 					{% endif %}
-					{% if ( header_button_title ) %}
-						<div class="row clearfix">
-							<div class="col-md-4">
-								<a href="{{ header_button_link }}" class="btn btn-small btn-medium btn-primary btn-block">{{ header_button_title }}</a>
-							</div>
-						</div>
+
+					{% if ( campaigns ) %}
+						{% for campaign in campaigns %}
+							<a class="tag-item " href="{{ campaign.link }}">#{{ campaign.name|e('wp_kses_post')|raw }}</a>
+						{% endfor %}
 					{% endif %}
 				</div>
-			</div>
+			{% endif %}
+			<h1 class="page-header-title">{{ header_title|e('wp_kses_post')|raw }}</h1>
+			{% if ( header_subtitle ) %}
+				<h3 class="page-header-subtitle">{{ header_subtitle }}</h3>
+			{% endif %}
+			{% if ( header_description ) %}
+				<p class="page-header-content">{{ header_description|e('wp_kses_post')|raw }}</p>
+			{% endif %}
+			{% if ( header_button_title ) %}
+				<div class="row">
+					<div class="col-md-4">
+						<a href="{{ header_button_link }}" class="btn btn-primary btn-medium page-header-btn">{{ header_button_title }}</a>
```

Comment:
Maybe you can use default here as you did in issue, in case the link is not defined
` issue.link|default('#')`

Reply:
If the link is not defined, we should not be showing anything at all! (Add an IF block)